aspnetcore/fundamentals/environments/sample/EnvironmentsSample/Startup.cs

```diff
@@ -18,12 +20,26 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();
+            services.Configure<CookiePolicyOptions>(options =>
```

Comment:
Consider creating a method which both `ConfigureServices` and `ConfigureStagingServices` invoke. Move this code there, since it's identical in both methods.